$NetBSD: patch-makefile,v 1.9 2015/01/29 21:48:15 wiz Exp $

Link against pkgsrc versions of expat, zlib, flac, jpeg, and
sqlite3, and add appropriate linker search paths.

*nix: Fixed linking tools against system libraries
https://github.com/mamedev/mame/commit/b55b45a5a99b8bce694b7ed3cd2d4a00d790f6ae

--- makefile.orig	2015-01-29 21:11:10.000000000 +0000
+++ makefile
@@ -227,19 +227,19 @@ endif
 # BIGENDIAN = 1
 
 # uncomment next line to build expat as part of MAME build
-BUILD_EXPAT = 1
+#BUILD_EXPAT = 1
 
 # uncomment next line to build zlib as part of MAME build
-BUILD_ZLIB = 1
+#BUILD_ZLIB = 1
 
 # uncomment next line to build libflac as part of MAME build
-BUILD_FLAC = 1
+#BUILD_FLAC = 1
 
 # uncomment next line to build jpeglib as part of MAME build
-BUILD_JPEGLIB = 1
+#BUILD_JPEGLIB = 1
 
 # uncomment next line to build libsqlite3 as part of MAME/MESS build
-BUILD_SQLITE3 = 1
+#BUILD_SQLITE3 = 1
 
 # uncomment next line to build PortMidi as part of MAME/MESS build
 BUILD_MIDILIB = 1
@@ -739,7 +739,7 @@ DRIVLISTOBJ = $(OBJ)/$(TARGET)/$(SUBTARG
 #-------------------------------------------------
 
 # start with an empty set of libs
-LIBS = 
+LIBS = -Wl,-R${PREFIX}/lib -L${PREFIX}/lib
 
 # add expat XML library
 ifeq ($(BUILD_EXPAT),1)
@@ -756,6 +756,7 @@ INCPATH += -I$(3RDPARTY)/zlib
 ZLIB = $(OBJ)/libz.a
 else
 LIBS += -lz
+BASELIBS += -lz
 ZLIB =
 endif
 
@@ -766,6 +767,7 @@ FLAC_LIB = $(OBJ)/libflac.a
 # $(OBJ)/libflac++.a
 else
 LIBS += -lFLAC
+BASELIBS += -lFLAC
 FLAC_LIB =
 endif
 
